cmake/kokkos_enable_devices.cmake

```diff
@@ -41,6 +41,16 @@ ENDIF()
 KOKKOS_DEVICE_OPTION(OPENMP ${OMP_DEFAULT} HOST "Whether to build OpenMP backend")
 
 KOKKOS_DEVICE_OPTION(OPENACC OFF DEVICE "Whether to build the OpenACC backend")
+IF (KOKKOS_ENABLE_OPENACC)
+  COMPILER_SPECIFIC_FLAGS(
+    Clang -fopenacc -fopenacc-fake-async-wait
+          -Wno-openacc-and-cxx -Wno-openmp-mapping -Wno-unknown-cuda-version
+          -Wno-pass-failed
+  )
+  COMPILER_SPECIFIC_DEFS(
+    Clang KOKKOS_WORKAROUND_OPENMPTARGET_CLANG
```

Comment:
That macro is originally used by OpenMPTarget/Clang to disable unsupported unit tests in `core/unit_test/TestComplex.hpp`, and the same tests are disabled for OpenACC/Clacc too since Clacc is built on top of the same OpenMP implementation in Clang/LLVM. (Clacc internally performs OpenACC-to-OpenMP translation to use the existing OpenMP implementation in LLVM to support OpenACC.)